cli/standalone.rs

```diff
@@ -150,6 +151,28 @@ pub async fn create_standalone_binary(
     } else {
       output
     };
+
+  if output.exists() {
+    // If the output is a directory, throw error
+    if output.is_dir() {
+      let error_msg =
+        format!("Could not compile: {:?} is a directory.", &output);
+      return Err(generic_error(error_msg));
```

Comment:
Nitpick: this could be replaced by:
```rust
use deno_core::error::bail;
...
bail!("Could not compile: {:?} is a directory.", &output);
```

Using `generic_error` here is an overkill because this function never interacts with JS runtime (it's a CLI utility)

Reply:
I see. Done, thanks!